### ISSUE TITLE
Blog Posts Block: Posts Set to Formats Other than Standard Don't Respect Excerpt Length Setting

### ISSUE THREAD

**Issue by katiebethbrown** [Automattic/newspack-blocks#660]
**Describe the bug**
When blog posts are set to a post format other than Standard, the Blog Posts Block doesn't respect the excerpt length setting. 

**To Reproduce**
1. Use a theme that has post formats, like Prosperity
2. Create a new post, and set it to use another format like Aside.
3. Add the posts block to a new page
4. The block will show the full post regardless of the excerpt length setting. Changing the setting does not impact the block at all.

**Screenshots**
You can see that the excerpt is set to show the 55 word default, but the full posts continue to display.

<img width="800" alt="screenshot of blog posts block showing full posts with arrow pointing to the excerpt word count setting set to show 55 words" src="https://user-images.githubusercontent.com/42279181/103389975-41597700-4acf-11eb-8610-7d3f38be90a9.png">

**Expected behavior**
The posts displayed in the block should respect the excerpt length setting.

Possibly related to this change, as it was working as expected until recently: https://github.com/Automattic/newspack-blocks/pull/648

**Comment by kspilarski** [comment]
Similar case to 3839624-zd and I was also able to recreate the issue on my test site. 
Also, regardless of the theme in use (with Post format feature or not), in Blog Posts block, the "Max number of words in excerpt" length does not get applied.

**Comment by csabarakasz** [comment]
Seems like the same issue shows up in this support forum question as well: https://wordpress.com/forums/topic/latest-posts-block-cannot-set-excerpt-length/

**Comment by laurelfulford** [comment]
This is working as designed, though I fully acknowledge that it's confusing: the Aside post format is meant to show in full -- it also doesn't link back to the full post like the standard post type does. 

There's some more information in this issue here: https://github.com/Automattic/newspack-blocks/pull/591

We originally added Aside support for a very particular use case; I think at this point it might make more sense to just remove the Aside support (since it's confusing!) than trying to change how it works. 

I'm going to spin up a new issue to discuss.

**Comment by laurelfulford** [comment]
We ultimately decided to remove support for the Aside post format in the theme, and the different handling in the block in In #1139. Once this change is merged into WordPress.com, posts with the Aside format will be treated like Standard posts (displaying `the_excerpt()` rather than `the_content()`, and linking to the full post). 

Hopefully this helps avoid any future confusion! I'm going to close this issue, but please let us know if you have any questions at all about this change!
